src/dev/build/tasks/install_dependencies_task.ts

```diff
@@ -31,7 +31,7 @@ export const InstallDependencies: Task = {
       extraArgs: [
         '--production',
         '--ignore-optional',
-        '--frozen-lockfile',
+        '--pure-lockfile',
```

Comment:
Why the change from frozen?

Reply:
We are now using `link:` in the `package.json` that we later change to `file:` in the build process which end ups changing the `yarn.lock` file so we need that change (that we were also using before introducing the yarn workspaces).